flutter-winit/src/window.rs

```diff
@@ -192,31 +195,42 @@ impl FlutterWindow {
                         WindowEvent::CloseRequested => *control_flow = ControlFlow::Exit,
                         WindowEvent::Resized(_) => resize(&engine, &context),
                         WindowEvent::HiDpiFactorChanged(_) => resize(&engine, &context),
-                        WindowEvent::CursorEntered { .. } => {
-                            engine.send_pointer_event(
-                                FlutterPointerPhase::Add,
-                                cursor,
-                                FlutterPointerSignalKind::None,
-                                (0.0, 0.0),
-                                FlutterPointerDeviceKind::Mouse,
-                                FlutterPointerMouseButtons::Primary,
-                            );
+                        WindowEvent::CursorEntered { device_id, .. } => {
+                            let idx = device_id.get_index();
+                            eprintln!("{:?}", active_devices);
+                            match active_devices.get(&idx) {
```

Comment:
Once moved to `HashSet`, we can use just a single `insert` statement, and use the `bool` resort to see if the value already existed.